keyboards/cannonkeys/ortho48v2/ortho48v2.c

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2022 CannonKeys
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "ortho48v2.h"
+
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+        if (clockwise) {
+            tap_code_delay(KC_VOLU, 10);
+        } else {
+            tap_code_delay(KC_VOLD, 10);
+        }
+    return encoder_update_user(index, clockwise);
+}
```

Comment:
Don't need this at the keyboard level. Suggest adding encodermap to default and via keymaps instead.

```suggestion
```

Reply:
Thanks for the tip here! Removed this and added encoder map instead - should be all set now

Reply:
> Don't need this at the keyboard level. Suggest adding encodermap to default and via keymaps instead.

Should still be added to the keyboard's c file, until such time that the configurator supports encoders, so that there is default behavior for them, and users aren't left with useless hardware.